typescript/packages/subsurface-viewer/src/components/Map.tsx

```diff
@@ -1689,10 +1689,11 @@ function updateViewState(
     camera: ViewStateType,
     boundingBox: BoundingBox3D,
     size: Size,
-    is3D = true
+    is3D = true,
+    verticalScale = 1
 ): ViewStateType {
     if (isCameraDefined(camera)) {
-        return camera;
+        return scaleCameraZoom(camera, verticalScale, is3D);
```

Comment:
I'm a bit concerned about the impact on memos.
I fear that returning a modified version of the camera here will force re-rendering... :/

I tested it locally, and /story/subsurfaceviewer-examples-camera--synced-subsurface-viewers seems to be broken (at least compared to published storybook).